cmd/controller/controller.go

```diff
@@ -430,8 +431,18 @@ func (c *command) start(ctx context.Context) error {
 		c.ClusterComponents.Add(ctx, metrics)
 	}
 
-	if !slices.Contains(c.DisableComponents, constant.KubeletConfigComponentName) {
-		c.ClusterComponents.Add(ctx, controller.NewKubeletConfig(c.K0sVars, adminClientFactory))
+	if !slices.Contains(c.DisableComponents, constant.WorkerConfigComponentName) {
```

Comment:
same suggestion as in https://github.com/k0sproject/k0s/pull/2425#discussion_r1033538534

Reply:
Done in the other PR.